.docker-hub/varnish/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG VERSION=7.5.0
```

Comment:
Mostly copied from https://github.com/softonic/varnish/blob/master/Dockerfile

As an alternative, we could also set this up as an independent package outside of this repository (e.g. ecamp/varnish) and regularly build/deploy on docker hub. Both is ok for me.